Add warning related to MQ configuration (#3516)
magento/devdocs: guides/v2.2/extension-dev-guide/message-queues/config-mq.md

| exchange             | The name of the exchange to publish to. The default system exchange name is `magento`. |
| disabled             | Determines whether this queue is disabled. The default value is `false`. |

{: .bs-callout .bs-callout-warning }

> Be aware, that for each `topic` only one `publisher` can be enabled at the same time

Reviewer: ```suggestion
You cannot enable more than one `publisher` for each `topic`.
```
